src/repositories/tasks.ts

```diff
@@ -0,0 +1,41 @@
+import { OAuth1Client } from '@extrahorizon/javascript-sdk';
+import { permissionModes } from '../commands/tasks/taskConfig';
+
+export interface FunctionCreation {
+  name: string;
+  description: string;
+  entryPoint: string;
+  code: string;
+  runtime: string;
+  timeLimit?: number;
+  memoryLimit?: number;
+  environmentVariables?: Record<string, any>;
+  executionOptions?: {
+    permissionMode: permissionModes;
+  };
+  retryPolicy?: {
+    enabled: boolean;
+    errorsToRetry: string[];
+  };
+}
+
+export const functions = ({
```

Comment:
Tis' a real shame the SDK calls aren't available for this yet, would of made this whole process quite a bit faster 😓